src/__tests__/__fixtures__/interfaces.ts

```diff
@@ -4,12 +4,6 @@
  * This file contains dummy code for testing the typescript plugin.
  */
 
-export enum Intent {
-    Primary,
-    Success,
-    Warning,
-    Danger,
-}
 export type IconName = "add" | "remove" | "plus" | "minus";
```

Comment:
You move the test for the `Intent` enum to `enums.ts` but you left this unioned string type here and duplicated it in `enums.ts`

Reply:
yes i moved all the enums to a separate test but i left just the union here so we're testing a non-basic type